src/Core/Models/Business/SubscriptionUpdate.cs

```diff
@@ -30,29 +29,23 @@ public bool UpdateNeeded(Subscription subscription)
         planId == null ? null : subscription.Items?.Data?.FirstOrDefault(i => i.Plan.Id == planId);
 }
 
-public class SeatSubscriptionUpdate : SubscriptionUpdate
+public abstract class BaseSeatSubscriptionUpdate : SubscriptionUpdate
 {
     private readonly int _previousSeats;
-    private readonly StaticStore.Plan _plan;
+    protected readonly StaticStore.Plan _plan;
```

Comment:
`protected` properties name should be in PascalCase, in this case it should be 
```suggestion
    protected readonly StaticStore.Plan Plan;
```